### PR TITLE
18 00 transactions example

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR adds the transactions example to the documentation.

### CODE REVIEW

File: docs/transactions/transactions-example.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+This page shows an example of how one might use transactions in a workflow.
+
+## Create a Transaction
+
+A transaction is created, and collections coll1 and coll2 are expected to be written to within the transaction:
+
+```bash
+curl -X 'POST' \
+  'https://api-xxyyzz-san.eng.macrometa.io/_fabric/_system/_api/transaction/begin' \
```
Comment (luisleite-macrometa) [inline docs/transactions/transactions-example.md]: In this and the following `curl` commands, I'd use a more generic fake URL, like https://api-xxyyzz.macrometa.io/...

File: docs/transactions/transactions-example.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+---
+sidebar_position: 100
+title: Transactions Example
+---
+
+This page shows an example of how one might use transactions in a workflow.
+
+## Create a Transaction
+
+A transaction is created, and collections coll1 and coll2 are expected to be written to within the transaction:
```
Comment (luisleite-macrometa) [inline docs/transactions/transactions-example.md]: Outside of code blocks, shouldn't we somehow highlight the collection names, like "coll1" and "coll2", or _coll1_ and _coll2_, or `coll1` and `coll2`?

File: docs/transactions/transactions-example.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  "collections": {
+    "write": [
+      "coll1",
+      "coll2"
+    ]
+  }
+}'
+```
+
+Here is a possible response, in which the ID of the just-created transaction is 14464588:
```
Comment (luisleite-macrometa) [inline docs/transactions/transactions-example.md]: Outside of code blocks, shouldn't we highlight the transaction ID, like "14464588", or _14464588_, or `14464588`?

File: docs/transactions/transactions-example.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    {
+        "name": "bar",
+        "value": 523
+    }
+    ]'
+```
+
+## Delete a Document
+
+Then a document with key abc-def-ghi is removed from collection coll2 within transaction 14464588:
```
Comment (luisleite-macrometa) [inline docs/transactions/transactions-example.md]: Outside of code blocks, shouldn't we highlight the document key, like "abc-def-ghi", or _abc-def-ghi_, or `abc-def-ghi`?

File: docs/transactions/transactions-example.md (hunk 2 above)
Comment (diana-macrometa) [reply docs/transactions/transactions-example.md]: Yes, good catch.

File: docs/transactions/transactions-example.md (hunk 3 above)
Comment (diana-macrometa) [reply docs/transactions/transactions-example.md]: Honestly, it's kind of a gray area. I'm going to highlight it to reduce confusion, though.

**Review by luisleite-macrometa — APPROVED** [review]

**Pull request merged**